pennylane/templates/subroutines/phase_adder.py

```diff
@@ -178,7 +178,11 @@ def _unflatten(cls, data, metadata):
 
     def map_wires(self, wire_map: dict):
         new_dict = {
-            key: [wire_map.get(w, w) for w in self.hyperparameters[key]]
+            key: (
+                [wire_map.get(w, w) for w in self.hyperparameters[key]]
+                if self.hyperparameters[key][0] is not None
+                else None
+            )
```

Comment:
Why is the change to PhaseAdder needed?

Reply:
Qml.assert_valid_op, was doing a mapping of the decomposition wires. PhaseAdder was receiving [None] instead of None, which generated a bug in the decomposition